docs/install_guides/installing-openvino-pip.md

```diff
@@ -1,107 +1,142 @@
 
-### Step 1. Set Up Python Virtual Environment
+For system requirements and troubleshooting, see https://pypi.org/project/openvino/
 
-Use a virtual environment to avoid dependency conflicts. 
+Step 1. Set Up Python Virtual Environment
++++++++++++++++++++++++++++++++++++++++++
+
+Use a virtual environment to avoid dependency conflicts.
 
 To create a virtual environment, use the following command:
 
-@sphinxdirective
 
-.. tab:: Linux and macOS
+.. tab-set::
 
-   .. code-block:: sh
-   
-      python3 -m venv openvino_env
-   
-.. tab:: Windows
+    .. tab-item:: Linux and macOS
+       :sync: linmac
 
-   .. code-block:: sh
-   
-      python -m venv openvino_env
-     
-     
-@endsphinxdirective
+       .. code-block:: sh
 
-### Step 2. Activate Virtual Environment
+          python3 -m venv openvino_env
 
-@sphinxdirective
+    .. tab-item:: Windows
+       :sync: win
+
+       .. code-block:: sh
+
+          python -m venv openvino_env
 
-.. tab:: Linux and macOS
 
-   .. code-block:: sh
-   
-      source openvino_env/bin/activate
-   
-.. tab:: Windows
+Step 2. Activate Virtual Environment
+++++++++++++++++++++++++++++++++++++
+
+
+.. tab-set::
+
+    .. tab-item:: Linux and macOS
+       :sync: linmac
+
+       .. code-block:: sh
+
+          source openvino_env/bin/activate
+
+    .. tab-item:: Windows
+       :sync: win
+
+       .. code-block:: sh
+
+          openvino_env\Scripts\activate
 
-   .. code-block:: sh
-   
-      openvino_env\Scripts\activate
 
 .. important::
 
    The above command must be re-run every time a new command terminal window is opened.
 
-@endsphinxdirective
 
-### Step 3. Set Up and Update PIP to the Highest Version
+Step 3. Set Up and Update PIP to the Highest Version
+++++++++++++++++++++++++++++++++++++++++++++++++++++
 
 Use the following command:
-```sh
-python -m pip install --upgrade pip
-```
 
-### Step 4. Install the Package
+.. code-block: sh
+
+   python -m pip install --upgrade pip
+
+
+Step 4. Install the Package
++++++++++++++++++++++++++++
 
 Use the following command:
-```
-pip install openvino
-```
 
-### Step 5. Verify that the Package Is Installed
+.. code-block: sh
+
+   pip install openvino
+
+
+Step 5. Verify that the Package Is Installed
+++++++++++++++++++++++++++++++++++++++++++++
 
 Run the command below:
-```sh
-python -c "from openvino.runtime import Core"
-```
+
+.. code-block: sh
+
+   python -c "from openvino.runtime import Core"
+
 
 If installation was successful, you will not see any error messages (no console output).
 
-Congratulations! You finished installing OpenVINO Runtime. Now you can start exploring OpenVINO's functionality through Jupyter Notebooks and sample applications. See the <a href="#whats-next">What's Next</a> section to learn more!
+Congratulations! You finished installing OpenVINO Runtime. Now you can start exploring OpenVINO's functionality through Jupyter Notebooks and sample applications. See the :ref:`What's Next <whats-next>`section to learn more!
```

Comment:
This part is not rendered correctly on Windows 11 Enterprise Chrome Version 111.0.5563.65 and macOS Catalina version 10.15.7 Safari 15.6.1 (15613.3.9.1.16, 15613):
<img width="710" alt="image" src="https://user-images.githubusercontent.com/107121967/224130097-c2eb9701-f76a-449c-87a4-01d841ba0e0b.png">

Reply:
Fixed :)